package.json

```diff
@@ -39,6 +40,7 @@
   },
   "scripts": {
     "audit": "npm audit | grep -oE 'https?\\:\\/\\/(www\\.)?(nodesecurity\\.io|npmjs\\.com)\\/advisories\\/[[:digit:]]+' | rev | cut -d '/' -f 1 | rev | diff known-vulns.txt -",
+    "cspell": "cspell **/*.css **/*.js",
```

Comment:
scpell is really helpful, shall we add it into `pre-commit` hook? So that we don't need another commit to fix spelling issue.

Reply:
It has "suggestions" in the VS Code plug-in, but I don't think it has any auto-fix functionality from the CLI. It could be used in the pre-commit to stop a commit with spelling issues though

Reply:
addressed in https://github.com/w3c/specberus/pull/990/commits/ce05ad8b405c24d1e03c0b6b2495f6d2238b952d